locale/templates/settings.ftl

```diff
@@ -479,7 +487,7 @@ row-defaults-status = None
 
 rk-header = Recovery key
 rk-enabled = Enabled
-rk-not-set = Not set
+rk-not-set = Not Set
```

Comment:
Why is this capitalized? The change won't impact l10n if the string is localized.

Reply:
As mentioned in a previous PR, this is just for our tests.  No change to L10n necessary